app/views/layouts/formio.html.haml

```diff
@@ -3,9 +3,10 @@
   %head
     %meta{ "http-equiv" => "content-type", "content" => "text/html; charset=UTF-8" }
     = render_view_hook "head"
-    = stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css", media: "all"
-    = stylesheet_link_tag "https://unpkg.com/formiojs@3.28.0/dist/formio.full.min.css", media: "all"
-    = javascript_include_tag "https://unpkg.com/formiojs@3.28.0/dist/formio.full.min.js"
+    - # using resource links based on https://github.com/formio/formio.js#form-building
```

Comment:
I think this technically should be `-#` for a haml comment

Reply:
TIL 🎉

Reply:
I wonder if HAML actually parses them differently

Reply:
Comments are likely stripped earlier, whereas non-haml-comments ruby comments have to go through the ruby parser, and so is presumably slower to get the final result